.github/workflows/heroku.yml

```
@@ -39,6 +39,17 @@ jobs:
       run: |
         # Even if comprehensive tests fail, core routes must pass
         python scripts/run_tests.py routes
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)